### PR TITLE
Updated Table Console helper for spanning cols and rows

### PR DESCRIPTION
**Pull request by hiddewie**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes, symfony/symfony#13438 |
| Applies to | 2.7+ |
| Fixed tickets | #5119 |

### CODE REVIEW

File: components/console/helpers/table.rst [hunk 1]
```diff
@@ -143,3 +143,91 @@ Here is a full list of things you can customize:
         $table->setStyle('colorful');
 
     This method can also be used to override a built-in style.
+
+Spanning multiple columns and rows
+----------------------------------
+
```
Comment (stof) [inline components/console/helpers/table.rst]: missing ``versionadded`` directive saying in which version this is available

**Comment by hiddewie** [comment]
I've added a `versionadded` clause.

File: components/console/helpers/table.rst [hunk 2]
```diff
@@ -143,3 +143,94 @@ Here is a full list of things you can customize:
...
+    $table->render();
+
+which results in:
+
+.. code-block:: text
+
+    +-----------------------+----------------------+-----------------+
+    | ISBN                  | Title                | Author          |
+    +-----------------------+----------------------+-----------------+
+    | This ISBN is spanning | A Tale of Two Cities | Charles Dickens |
```
Comment (javiereguiluz) [inline components/console/helpers/table.rst]: Sorry for being picky, but having two different books with the same ISBN doesn't make sense to me. What if we update this example to have two differente books (different ISBNs and Titles) written by the same author (that's the "rowspan")?

**Comment by javiereguiluz** [comment]
@hiddewie :+1: apart from my comment, I like your contribution a lot. Thanks!

File: components/console/helpers/table.rst (hunk 2 above)
Comment (hiddewie) [inline components/console/helpers/table.rst]: Good idea. I've changed the rowspan to use the author column.

**Comment by weaverryan** [comment]
You just helped us close an issue :) - you rock @hiddewie.

Thanks!

**Comment by hiddewie** [comment]
Thank you for all the positive feedback!

**Pull request merged**